src/client/src/orangehrmRecruitmentPlugin/components/VacancyDropdown.vue

```diff
@@ -37,6 +37,11 @@ export default {
       required: false,
       default: null,
     },
+    excludeInterviewers: {
```

Comment:
not clear what this is? this is vacancy dropdown, whats the relation with Interviewers?